apps/address_registry/urls.py

```diff
@@ -1,11 +1,19 @@
 from django.urls import include, path, re_path
+from rest_framework.routers import DefaultRouter
 
 from apps.address_registry.views import (
+    ContinentCountrySettlementViewSet,
+    DocumentViewSet,
     GenerateTestData,
     cities_application_json,
     cities_application_soap,
 )
 
+router = DefaultRouter()
+router.register(r"settlements", ContinentCountrySettlementViewSet, basename="nested-settlements")
+router.register(r"documents", DocumentViewSet, basename="document")
```

Comment:
I think we should stay consistent and either do singular/plural basenames
```suggestion
router.register(r"documents", DocumentViewSet, basename="documents")
```

Reply:
true, thanks